webapp/package.json

```diff
@@ -18,63 +18,68 @@
   "author": "",
   "license": "",
   "devDependencies": {
-    "@babel/cli": "7.5.5",
+    "@babel/cli": "7.15.4",
     "@babel/core": "7.23.2",
-    "@babel/plugin-proposal-class-properties": "7.5.5",
-    "@babel/plugin-proposal-object-rest-spread": "7.5.5",
-    "@babel/plugin-syntax-dynamic-import": "7.2.0",
-    "@babel/polyfill": "7.4.4",
-    "@babel/preset-env": "7.5.5",
-    "@babel/preset-react": "7.0.0",
-    "@babel/preset-typescript": "7.3.3",
-    "@babel/runtime": "7.4.5",
-    "@types/enzyme": "3.10.3",
+    "@babel/plugin-proposal-class-properties": "7.14.5",
+    "@babel/plugin-proposal-object-rest-spread": "7.14.5",
+    "@babel/plugin-syntax-dynamic-import": "7.8.3",
+    "@babel/polyfill": "7.12.1",
+    "@babel/preset-env": "7.15.4",
+    "@babel/preset-react": "7.14.5",
+    "@babel/preset-typescript": "7.15.0",
+    "@babel/runtime": "7.15.4",
+    "@types/enzyme": "3.10.18",
     "@types/jest": "27.5.1",
-    "@types/node": "12.7.0",
-    "@types/react": "16.8.24",
-    "@types/react-dom": "16.8.5",
-    "@types/react-intl": "3.0.0",
-    "@types/react-redux": "7.1.2",
-    "@types/react-router-dom": "4.3.4",
-    "@types/react-select": "3.0.4",
-    "@types/react-transition-group": "4.2.2",
-    "@typescript-eslint/eslint-plugin": "2.1.0",
-    "@typescript-eslint/parser": "2.34.0",
-    "babel-eslint": "10.0.2",
-    "babel-jest": "24.8.0",
+    "@types/node": "16.18.96",
+    "@types/react": "17.0.38",
+    "@types/react-dom": "17.0.11",
```

Comment:
I think `react`, `react-dom`, `@types/react`, and `@types/react-dom` should always be made to equal each other. I wonder why we're updating types here but not `react` itself? I know there is the challenge with `enzyme` there, but why are the types being updated?